Telerik.JustMock/AutoMock/MockingContainer.cs

```diff
@@ -183,32 +184,75 @@ public FuncExpectation<object> Arrange<TInterface>(Expression<Func<TInterface, o
 			return ProfilerInterceptor.GuardInternal(() => this.Get<TInterface>().Arrange(expression));
 		}
 
-		/// <summary>
+        /// <summary>
+        /// Entry-point for setting expectations.
+        /// </summary>
+        /// <typeparam name="TInterface">Mocking interface</typeparam>
+        /// <param name="expression">Target expression</param>
+        /// <returns>
+        /// Reference to <see cref="FuncExpectation<Task>"/> to setup the mock.
+        /// </returns>
+        public FuncExpectation<Task> Arrange<TInterface>(Expression<Func<TInterface, Task>> expression)
+        {
+            return ProfilerInterceptor.GuardInternal(() => this.Get<TInterface>().Arrange(expression));
+        }
+
+        /// <summary>
 		/// Entry-point for setting expectations.
 		/// </summary>
-		/// <typeparam name="TInterface">
-		/// Mocking interface
-		/// </typeparam>
+		/// <typeparam name="TInterface">Mocking interface</typeparam>
+		/// <typeparam name="TResult">Task result type</typeparam>
```

Comment:
Mixed spaces and tabs throughout the changes. Please unify.

Reply:
I will create a new task to reformat the whole project.